crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/execution/entry_point.rs

```diff
@@ -33,12 +33,29 @@ pub fn execute_call_entry_point(
     resources: &mut ExecutionResources,
     context: &mut EntryPointExecutionContext,
 ) -> EntryPointExecutionResult<CallInfo> {
+    let cheated_data = if let CallType::Delegate = entry_point.call_type {
+        runtime_state
+            .cheatnet_state
+            .trace_data
+            .current_call_stack
+            .top_cheated_data()
+            .clone()
+    } else {
```

Comment:
So we just don't want to treat the library call as calling another entrypoint? It goes through the Starknet OS layer, but I guess it makes sense? cc @Arcticae